src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceAfterCheck.java

```diff
@@ -91,6 +91,32 @@
  * <pre>
  * &lt;module name=&quot;NoWhitespaceAfter&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test {
+ *
+ *    public void lineBreak(String x) {
+ *     Integer.
+ *     parseInt(x); // Ok
```

Comment:
add 2 spaces to indent this line to right, to show that it is line wrap of previos line.